cran-comments.md

```diff
@@ -1,7 +1,10 @@
 ## Resubmission
```

Comment:
@mx-iao : I'd suggest include all old re-submission comments something like below so they know it's not the very first submission and we are keeping fixing things (in case it's assigned to a new reviewer again) 

Resbumission 2019-11-06 (latest)
...
Resbumission 2019-10-25
...
Resbumission 2019-10-01
...

Reply:
Great point. I'll do that

Reply:
Updated to add all resubmission history